halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/NodeFilter.java

```diff
@@ -64,17 +64,37 @@ public NodeFilter withAnyMaster() {
     return this;
   }
 
-  public static NodeFilter makeRejectAllFilter() {
-    return new NodeFilter();
+  public static NodeFilter makeAcceptAllFilter() {
```

Comment:
Seems like you need a `NodeFilterFactory` or `NodeFilterBuilder` alongside the actual `NodeFilter`.